A-4E-C/Cockpit/Scripts/Systems/afcs2.lua

```diff
@@ -910,7 +910,12 @@ function update_afcs()
         end
         return
     elseif afcs_state == AFCS_STATE_ATTITUDE_ONLY then
-        afcs_hold_bank(afcs_bank_angle_hold)
+        if afcs_bank_angle_hold > 5 or afcs_bank_angle_hold < -5 then
```

Comment:
This is a change that should be made in the transition station function on line 604.

```lua
if math.abs(math.deg(sensor_data.getRoll())) > 5 then
    afcs_bank_hold = 0.0
end
```

Reply:
This way the code that handles the AFCS behaviour in a given state can remain the same. As this is specifically a phenomenon that occurs on a state transition.